README.md

```diff
@@ -1 +1,28 @@
-# frontend-functional-tests
\ No newline at end of file
+Functional testing for frontend
+
+## Usage
+
+To run the tests, use the following command
+```
+node index.js
+```
+
+All results will be saved in `status.csv` with the following headers:
+`id,name,page_status,page_title,dataset_title,readme,csv_links,csv_preview_links,json_links,zip_links,datapackage_json,total_load_time,script_laod_time,layout_load_time,tables,graphs`
```

Comment:
Think this would look better if they are bullet points

Reply:
@zelima you mean, all the headers?

Reply:
fixed